bip-0388.mediawiki

```diff
@@ -57,6 +60,18 @@ The hardware signing device must guarantee that the user knows precisely what "p
 
 This makes it impossible for an attacker to surreptitiously modify the policy, therefore stealing or burning the user's funds.
 
+==== Avoiding key reuse ====
+
+Reusing public keys within a Script is a source of malleability when using miniscript policies, which has potential security implications.
```

Comment:
Capital-S Script usually refers to the Script language, perhaps go lowercase here:

```suggestion
Reusing public keys within a script is a source of malleability when using miniscript policies, which has potential security implications.
```